### ISSUE TITLE
Minimap resoultion

### ISSUE THREAD

**Issue by KuMoNDev** [Xxpromw3mtxX/cosmo_hud#17]
When changing different resoultions the outline on around the map does not correct its self

**Comment by KuMoNDev** [comment]
![image](https://user-images.githubusercontent.com/101035808/156912466-64543b36-d1a3-4381-976b-87e9526c9e81.png)

**Comment by Xxpromw3mtxX** [comment]
Yeah I know, this is a problem related to the fact that it isn't responsive.
This should be easy to solve but don't have the monitors to test it.

**Comment by KuMoNDev** [comment]
I can help you if you want, I have multiple monitors and wide monitors let me know

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Xxpromw3mtxX** [comment]
For sure!
If you'd like to help you can!
When you've tested it just open a PR and I'll merge it after a manual control.

**Comment by marcreiner** [comment]
Maybe this code snippet could help:
```LUA

function UpdateMinimapLocation()
  Citizen.CreateThread(function()
    -- Get screen aspect ratio
    local ratio = GetScreenAspectRatio()

    -- Default values for 16:9 monitors
    local posX = -0.0045
    local posY = 0.002

    if tonumber(string.format("%.2f", ratio)) >= 2.3 then
      -- Ultra wide 3440 x 1440 (2.39)
      -- Ultra wide 5120 x 2160 (2.37)
      posX = -0.185
      posY = 0.002
      print('Detected ultra-wide monitor, adjusted minimap')
    else 
      posX = -0.0045
      posY = 0.002
    end

    SetMinimapComponentPosition('minimap', 'L', 'B', posX, posY, 0.150, 0.188888)
    SetMinimapComponentPosition('minimap_mask', 'L', 'B', posX + 0.0155, posY + 0.03, 0.072, 0.162)
    SetMinimapComponentPosition('minimap_blur', 'L', 'B', posX - 0.0255, posY + 0.02, 0.18, 0.22)

    DisplayRadar(false)
    SetRadarBigmapEnabled(true, false)
    Citizen.Wait(0)
    SetRadarBigmapEnabled(false, false)
    DisplayRadar(true)
  end)
end
```


credits to: https://forum.cfx.re/t/mini-map-with-high-resolution-fix/4754680/6?u=mrcrnr
